src/AnchoredOverlay/AnchoredOverlay.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import React, {useCallback, useRef, useState} from 'react'
+import {Args, Meta} from '@storybook/react'
+
+import {BaseStyles, Box, ThemeProvider} from '..'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import {Button} from '../Button'
+
+export default {
+  title: 'Components/AnchoredOverlay',
+  component: AnchoredOverlay,
+  args: {
+    'With Anchor?': true,
+    width: 'small',
+    height: 'auto',
+    portalContainerName: 'Portal name',
+    side: 'outside-bottom',
+  },
+  argTypes: {
+    'With Anchor?': {
+      type: 'boolean',
+    },
+    width: {
+      type: {
+        name: 'enum',
+        value: ['small', 'medium', 'large', 'xlarge', 'xxlarge', 'auto'],
+      },
+      portalContainerName: {type: 'string'},
+    },
+    height: {
+      type: {
+        name: 'enum',
+        value: ['xsmall', 'small', 'medium', 'large', 'xlarge', 'auto', 'initial'],
+      },
+      portalContainerName: {type: 'string'},
+    },
+    side: {
+      type: {
+        name: 'enum',
+        value: [
+          'inside-top',
+          'inside-bottom',
+          'inside-left',
+          'inside-right',
+          'inside-center',
+          'outside-top',
+          'outside-bottom',
+          'outside-left',
+          'outside-right',
+        ],
+      },
+    },
+  },
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
+} as Meta
+
+export const Default = () => {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <AnchoredOverlay
+      open={open}
+      onOpen={() => setOpen(true)}
+      onClose={() => setOpen(false)}
+      renderAnchor={props => <Button {...props}>Button</Button>}
+    >
+      <Box width="100%" height="100%" backgroundColor="thistle" display="flex" flexDirection="column">
+        <img src={`//placekitten.com/200/300`} alt="kitten" />
```

Comment:
This made me laugh lol. Should we use a more generic placeholder instead of kittens as we make this more visible? The kitten part feels very dev-only / for testing instead of as documentation.

Reply:
Hey Josh, it looks like I came up with this, but it was actually from [9/1/21](https://github.com/primer/react/commit/83b888f0b325f9370ede9c5d7ac8f70702d6e8ea). Since it's been here for 2 years, it's your call if it's too dev-only. What's your stance?

Reply:
I think we could do something like a fake "hover card" to make it feel more realistic.

<img width="433" alt="Profile hover card" src="https://github.com/primer/react/assets/2313998/a9bb9e7b-7b87-46e8-b4aa-cc503419fcd0" />

Reply:
@green6erry makes sense! Totally get what you mean, those existing stories have been around for a while and they're just showing up here since you're moving stuff around.

I'm not sure what the expectations are for those stories under "behaviors", like Anchored Overlay. When looking at them, they look to be dev-only in that they're testing scenarios instead of serving as a reference for how to use the component.

I think if there is a quick way to simulate something like the hover card that'd be great but if not then we can always bring this in and change later 👍